examples/python/camera_video_stream/camera_video_stream.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import argparse
+import platform
+import subprocess
+
+import av
+import rerun as rr
+
+
+def setup_camera_input(video_device: str | None = None) -> av.container.InputContainer:
+    """
+    Setup the camera input container.
+
+    Uses first available video device. Exact behavior is platform dependent.
+    """
+    if platform.system() == "Darwin":
+        if video_device is None:
+            video_device = "0"
+
+        return av.open(
+            video_device,
+            format="avfoundation",
+            container_options={"framerate": "30"},  # `avfoundation` fails if the framerate is not set.
+        )
+    elif platform.system() == "Windows":
+        if video_device is None:
+            # On windows we *have* to know the device name. Query ffmpeg for it.
+            list_devices_cmd = ["ffmpeg", "-f", "dshow", "-list_devices", "true", "-i", "dummy", "-hide_banner"]
+            devices_output = subprocess.check_output(list_devices_cmd, stderr=subprocess.STDOUT).decode()
+
+            # Extract video device names
+            video_devices = []
+            for line in devices_output.split("\n"):
+                if '"' in line and "Alternative name" not in line:
```

Comment:
When I tested this on my Windows machine it got confused with my headphones:

```
Using video device: Microphone (HyperX Cloud II Wireless)
Could not enumerate video devices (or none found).

av.error.OSError: [Errno 5] I/O error: 'video=Microphone (HyperX Cloud II Wireless)'; last error log: [dshow] Could not enumerate video devices (or none found).
```

I tried sidestepping the whole `ffmpeg` shenanigans, but I don't think it's worth more time 😬

To fix it, I suggest we do:

```suggestion
                # FFmpeg will return both `(audio)` and `(video)` devices
                if "(video)" in line and "Alternative name" not in line:
```

Reply:
thanks, works on my windows as well!